modules/consignment-api/database.tf

```diff
@@ -0,0 +1,59 @@
+resource "random_password" "password" {
```

Comment:
Just wondering - do you know how we can do password rotation without API downtime? Might be worth adding to the tech backlog.

Reply:
I've added it to the github project and put a comment on the card with what you suggested about creating a second user.